web-marketplace/src/lib/wallet/wallet.tsx

```diff
@@ -192,6 +194,7 @@ export const WalletProvider: React.FC<React.PropsWithChildren<unknown>> = ({
       value={{
         wallet,
         walletConfig: walletConfigRef.current,
+        isWalletConnect: walletConnect,
```

Comment:
best to use the existing var `loginDisabled`, which also takes into account keplr mobile users